src/Microsoft.AspNetCore.Authentication.Twitter/TwitterHandler.cs

```diff
@@ -80,16 +78,9 @@ public TwitterHandler(IOptionsSnapshot<TwitterOptions> options, ILoggerFactory l
                 return HandleRequestResult.Fail("Missing or blank oauth_verifier");
             }
 
-            var cookieOptions = new CookieOptions
-            {
-                HttpOnly = true,
-                SameSite = SameSiteMode.Lax,
-                Secure = Request.IsHttps
-            };
-
-            Options.ConfigureStateCookie?.Invoke(Context, cookieOptions);
+            var cookieOptions = Options.StateCookie.Build(Context, Clock.UtcNow);
 
-            Response.Cookies.Delete(StateCookie, cookieOptions);
+            Response.Cookies.Delete(Options.StateCookie.Name, cookieOptions);
```

Comment:
I'm trying to think if a user would Build their options any differently if they knew they were building a delete cookie.